db/migrate/20250526095219_add_deactivation_fields_to_endpoints.rb

```diff
@@ -0,0 +1,9 @@
+class AddDeactivationFieldsToEndpoints < ActiveRecord::Migration[8.0]
+  disable_ddl_transaction!
```

Comment:
Do we need this in all migration files? Because of concurrent index additions but why do we need the concurrency? (just curious)

Reply:
I've added a section to the PR's description describing the migrations with indexes